.storybook/config.js

```diff
@@ -5,10 +5,8 @@ import 'babel-polyfill'
 setAddon(JSXAddon)
 
 function loadStories() {
-  const stories = require.context('../stories', true, /\.stories\.js$/)
+  const stories = require.context('..', true, /\.stories\.js$/)
```

Comment:
it turns out that this hits node_modules as well (where storybook itself has a bunch of dist files with `.stories.js` extensions), so it's probably not ideal 😅

I think the best solution is just to move our miscellaneous stories folder under `src`, since we're colocating anyway